src/webview/html/messageReactionAsHtml.js

```diff
@@ -19,5 +19,5 @@ export default (
     realmEmoji[reaction.name]
       ? getRealmEmojiHtml(realmEmoji[reaction.name])
       : emojiMap[reaction.name]
-  } ${reaction.count}
+  }&#160;${reaction.count}
```

Comment:
Pretty good fix!

Reply:
No particular reason - agreed that &nbsp would be more clear. I'll change that :)